CommandHelper/LocalPackages/commands/chars/geniealaddin.yml

```diff
@@ -35,5 +35,8 @@ powers:
 - /normalsize
 - /carryplayer <player>
 - /dropplayer
+- /tp <player>
+- /empower <player>
+- /disguise
```

Comment:
don't forget to give him the permissions too

Reply:
the permissions for the powers ur tryin to add

Reply:
HUH

Reply:
You need to add permissions so he can use the powers